src/pages/container/EventsTable/util.tsx

```diff
@@ -2,6 +2,16 @@ import { Link } from "react-router-dom";
 import { getTaskRoute } from "constants/routes";
 import { PodEventsQuery } from "gql/generated/types";
 import { Unpacked } from "types/utils";
+import { errorReporting } from "utils";
+
+const { reportError } = errorReporting;
+
+enum EventTypes {
```

Comment:
We store Host types in the `types/host` directory can we do the same here?
https://github.com/evergreen-ci/spruce/blob/7ea19bbd4ea941d9833c1a53b0ed0138312c0f1f/src/types/host.ts#L37